include/klee/Support/ModuleUtil.h

```diff
@@ -41,18 +41,19 @@ linkModules(std::vector<std::unique_ptr<llvm::Module>> &modules,
 
 #if defined(__x86_64__) || defined(__i386__)
 #define addFunctionReplacement(from, to)                                       \
-  {#from "f", #to "f"}, {#from, #to}, { #from "l", #to "l" }
+  {#from "f", #to "f"}, {#from, #to}, { "" #from "l", #to "l" }
```

Comment:
Are these empty quotes required here (and below)?

Reply:
without quotes clang-format split it to 3 lines